src/loader/loader-info.c

```diff
@@ -0,0 +1,2 @@
+#include <unistd.h>
+const ssize_t offset_to_pokedata_workaround=2248;
```

Comment:
This file should have been in `.gitignore` since it's introduction (Sorry, just added it d1fc3a6b0f34d06c9a301de86175e15fffc5dfb3, I haven't noticed earlier because I cross-compile using `termux-packages` which doesn't leave artifacts in git repository). You'll need to remove it.

Reply:
Ok.  Will do.